.textlintrc.json

```diff
@@ -16,7 +16,10 @@
         "/<http://.*>/",
         "medium.com/opentelemetry",
         "github.com/open-telemetry",
+        "`javaagent`",
```

Comment:
Why the code-fence ticks @svrnm?

Reply:
To elaborate: I don't think that the text linter checks words inside code fences. Let me check ... .